xrpl_api/src/api/subscribe.rs

```diff
@@ -28,6 +24,58 @@ pub struct SubscribeRequest {
     url_password: Option<String>,
 }
 
+/// A book on the ledger.
+#[derive(Debug, Clone, Serialize)]
+pub struct Book {
```

Comment:
Moved this here from its separate file since it only seems related to subscribe/unsubscribe